lib/kitchen/driver/oci.rb

```diff
@@ -35,55 +34,39 @@ module Driver
     #
     # @author Stephen Pearson <stephen.pearson@oracle.com>
     class Oci < Kitchen::Driver::Base # rubocop:disable Metrics/ClassLength
+      # required config items
       required_config :compartment_id
       required_config :availability_domain
-      required_config :image_id
       required_config :shape
       required_config :subnet_id
 
-      default_config :use_private_ip, false
-      default_config :oci_config_file, nil
-      default_config :oci_profile_name, nil
+      # common config items
+      default_config :instance_type, 'compute'
       default_config :hostname_prefix, nil
       default_keypath = File.expand_path(File.join(%w[~ .ssh id_rsa.pub]))
       default_config :ssh_keypath, default_keypath
       default_config :post_create_script, nil
       default_config :proxy_url, nil
       default_config :user_data, []
       default_config :freeform_tags, {}
+
+      # compute config items
+      default_config :image_id
+      default_config :use_private_ip, false
+      default_config :oci_config_file, nil
+      default_config :oci_profile_name, nil
       default_config :setup_winrm, false
       default_config :winrm_user, 'opc'
       default_config :winrm_password, nil
       default_config :use_instance_principals, false
 
```

Comment:
Maybe we should add a `default_config :dbaas, {}` here, just to show people who look at the source code that the parameter exists.  And perhaps a comment to direct people to look at the `dbaas_launch_details` function if they want to see what the default values for dbaas are?